docs/advanced/meta.rst

```diff
@@ -0,0 +1,130 @@
+.. _`advanced-meta`:
+
+============================
+Passing information via Meta
+============================
+
+In some cases, Page Objects might require additional information to be passed to
+them. Such information can dictate the behavior of the Page Object or affect its
+data entirely depending on the needs of the developer.
+
+If you can recall from the previous basic tutorials, one essential requirement of
+Page Objects that inherit from :class:`~.WebPage` or :class:`~.ItemWebPage` would
+be :class:`~.HttpResponse`. This holds the HTTP response information that the
+Page Object is trying to represent.
+
+In order to standardize how to pass arbitrary information inside Page Objects,
+we'll need to use :class:`~.Meta` similar on how we use :class:`~.HttpResponse`
+as a requirement to instantiate Page Objects:
+
+.. code-block:: python
+
+    import attr
+    import web_poet
+
+    @attr.define
+    class SomePage(web_poet.ItemWebPage):
+        # The HttpResponse attribute is inherited from ItemWebPage
+        meta: web_poet.Meta
+
+    response = web_poet.HttpResponse(...)
+    meta = web_poet.Meta("arbitrary_value": 1234, "cool": True)
```

Comment:
it is a correct syntax?

Reply:
Thanks for catching! I was missing the `{` and `}` characters here. Updated this on https://github.com/scrapinghub/web-poet/pull/22/commits/f0a3419325508c4e8e4b5bfec87a1d19aaacc660#diff-f010592a6c5d89e2fe40fc8b7a02d88d657351a09fc8f693e91a2cf6bf445a83R30-R34